--- a/src/XMonad/Core.hs
+++ b/src/XMonad/Core.hs
@@ -433,7 +433,11 @@

 -- | Return the path to @~\/.xmonad@.
 getXMonadDir :: MonadIO m => m String
-getXMonadDir = io $ getAppUserDataDirectory "xmonad"
+getXMonadDir = io $ do e <- userExists; if e then user else (return "/etc/xmonad")
+  where
+  user = getAppUserDataDirectory "xmonad"
+  userExists = user >>= doesDirectoryExist
+

 -- | 'recompile force', recompile @~\/.xmonad\/xmonad.hs@ when any of the
 -- following apply:

